services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaServiceITest.kt

```diff
@@ -431,10 +431,10 @@ class HankeKayttajaServiceITest : DatabaseTest() {
                 prop(HankeKayttajaDto::sahkoposti).isEqualTo(email)
                 prop(HankeKayttajaDto::etunimi).isEqualTo("Joku")
                 prop(HankeKayttajaDto::sukunimi).isEqualTo("Jokunen")
-                prop(HankeKayttajaDto::nimi).isEqualTo("Joku Jokunen")
                 prop(HankeKayttajaDto::puhelinnumero).isEqualTo("0508889999")
                 prop(HankeKayttajaDto::kayttooikeustaso).isEqualTo(Kayttooikeustaso.KATSELUOIKEUS)
                 prop(HankeKayttajaDto::tunnistautunut).isFalse()
+                prop(HankeKayttajaDto::kutsuttu).isNotNull().isRecent()
```

Comment:
If I understand correctly `isRecent` assertion succeeds if the value is `null`. I wonder if that is the right way to assert the time? In some places, where `isRecent` is used there is no `isNotNull` assertions accompaning it and at least my intuition would add the null-check implicitely in `isRecent`. How can a null date/time be recent?

Reply:
Yes, I agree that `assertThat(null).isRecent()` should fail. I changed all the `isRecent` type asserts to fail with nulls.

I also thought about making them work only on non-null values, but that would cause unnecessary bloat when used.